Jenkinsfile-Agent-Test-3

```diff
@@ -77,9 +77,14 @@ pipeline {
                 }
                 stage('Group2') {
                     stages {
-                        stage('reserve stages') {
+                        stage('postgresql 9.2.x-9.4.x (36)') {
                             steps {
-                                echo "reserve."
+                                sh 'bash test/plugin/run.sh --build_id=${BUILD_ID} postgresql-scenario'
+                            }
+                        }
+                        stage('postgresql-9.4 (16)') {
```

Comment:
You miss this change. The name should be `postgresql-9.4.1207+`

Reply:
Your new name should be fine too.

Reply:
ACK